apps/nextjs/src/app/(dashboard)/dashboard/_components/sidebar/side-bar-client-components.tsx

```
@@ -0,0 +1,79 @@
+DropDownMenuContentMobile.displayName = "DropDownMenuContentMobile";
+
+
+export const ProjectsSidebarMenuAction = forwardRef<
+React.ComponentRef<typeof SidebarMenuAction>,
+React.ComponentPropsWithoutRef<typeof SidebarMenuAction>
+>(   ({ children, ...props}, ref) => {
+  return (
+    <SidebarMenuAction
+      ref={ref}
+      showOnHover
+      role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)